BackEnd/src/main/kotlin/quest/marketstack/tradingapp/controller/ClosedTradeController.kt

```diff
@@ -0,0 +1,17 @@
+@file:Suppress("ktlint:standard:no-wildcard-imports")
```

Comment:
This is just suppressing the warning from KtLint that says you can't import things with a wildcard (an asterisk at the end of the import, in this case `import org.springframework.web.bind.annotation.*`). Copy paste this wherever you get that warning to get rid of it